File: webapi/MyTrace/Models/Color.cs

using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace MyTrace.Models;

public partial class Color
{
    public string Id { get; set; } = null!;

    public int OrganizationId { get; set; }

    public string Color1 { get; set; } = null!;

    [JsonIgnore]
    public virtual ICollection<ModelsSizesColor> ModelsSizesColors { get; } = new List<ModelsSizesColor>();

    [JsonIgnore]
    public virtual Organization? Organization { get; set; } = null;
}

File: frontend/src/models/api/Color.ts
import { Pagination } from "./Pagination";

export interface ColorPagination extends Pagination {
  results: [
    {
      id: string;
      organizationId: number;
      color1: string;
    }
  ];
}

export type Color = {
  id: string;
  organizationId: number;
  color1: string;
};
